modules/dummy-module-abc.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+
+:_content-type: PROCEDURE
+[id="gathering-data-about-your-cluster-using-must-gather_{context}"]
+= Gathering data about your cluster using the must-gather command
+
+The Performance Profile Creator (PPC) tool requires `must-gather` data. As a cluster administrator, run the must-gather command to capture information about your cluster.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* The OpenShift CLI installed.
+
+.Procedure
+
+. Optional: Verify that a matching machine config pool exists with a label:
++
+[source,terminal]
+----
+oc describe mcp/worker-rt
+----
++
+[source,bash]
+----
+Name:         worker-rt
+Namespace:
+Labels:       machineconfiguration.openshift.io/role=worker-rt
+----
+
+. If a matching label does not exist add a label for a machine config pool (MCP) that matches with the MCP name:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc label mcp _<mcp-name>_ _<mcp_name>_=""
+----
+
+. Navigate to the directory where you want to store the `must-gather` data.
+
+. Collect cluster information by running the following command.
++
+[source,terminal]
+----
+oc adm must-gather
+----
+
+. (Optional)- Create a compressed file from the must-gather directory:
++
+[source,terminal]
+----
+$ tar cvaf must-gather.tar.gz must-gather/
+----
++
+[NOTE]
+====
+Compressed output is required if you are running the Performance Profile Creator wrapper script.
```

Comment:
```suggestion
Compressed output is required if you are running the Performance Profile Creator wrapper (PPC) script.
```

Reply:
When using the term for the first time in the module use the abbreviation in parenthesis.